cognite/client/_api/data_modeling/graphql.py

```diff
@@ -156,6 +156,27 @@ def query(self, id: DataModelIdentifier, query: str, variables: dict[str, Any] |
                 ...     id=("mySpace", "myDataModel", "v1"),
                 ...     query="listThings { items { thingProperty } }",
                 ... )
+
+            Execute a graphql with result paginated and returning a cursor to next page::
+
+                >>> query = '''
+                ...     listThings (
+                ...         first: 10
+                ...         after: '<previous endCursor>'
+                ...     ) {
+                ...         items {
+                ...             thingProperty
+                ...         }
+                ...         pageInfo {
+                ...             endCursor
+                ...             hasNextPage
```

Comment:
This `hasNextPage` is deprecated, should only rely on the existence of an `endCursor`!

Reply:
https://github.com/cognitedata/doc.cognitedata.com/pull/1914